metro.config.js

```diff
@@ -1,17 +1,15 @@
-/**
- * Metro configuration for React Native
- * https://github.com/facebook/react-native
- *
- * @format
- */
-
-module.exports = {
-  transformer: {
-    getTransformOptions: async () => ({
```

Comment:
pls leave this property for transformer as well

U can learn about metro configuration here - https://facebook.github.io/metro/docs/configuration/#gettransformoptions